docs/projects/pulsar_adc_pmdz/index.rst

```diff
@@ -0,0 +1,403 @@
+.. _pulsar_adc_pmdz:
+
+PULSAR ADC PMDZ HDL project
```

Comment:
Could you change the title using this [this](https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad9081_fmca_ebz/index.rst?plain=1#L3) format?

Reply:
Done.